ntbs-service/Pages/Shared/_NotificationLayout.cshtml

```diff
@@ -62,48 +65,48 @@
     {
         <nhs-grid-column grid-column-width="OneQuarter">
             <div>
-                <li class="@("contents-list-item " + ((bool)ViewData["IsPatientPage"] ? "contents-list-item--current" : ""))">
-                    <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="@Model.Notification.PatientEditPath">Person details</a>
-                </li>
-                <li class="@("contents-list-item " + ((bool)ViewData["IsEpisodePage"] ? "contents-list-item--current" : ""))">
+                <div class="@("contents-list-item " + ((bool)ViewData["IsPatientPage"] ? "contents-list-item--current" : ""))">

```

Comment:
I thin I'd be more tempted to warp this in a `ul` rather than change to `div`s.. is that option less straight forward (i imagine it might throw styling off at first)?

Reply:
Yeah with that route, would need to go in and strip out margin and/or padding. I can get this done if it's preferable though?

Reply:
that seems good, if it doesn't grow into yak-shaving too much ;)